README.rst

```diff
@@ -105,7 +105,7 @@ Run unit tests with:
       ``~/.datacube_integration.conf`` and edit to customise.
 
    - For instructions on setting up a password-less Postgres database, see
-      the `developer setup instructions <https://datacube-core.readthedocs.io/en/latest/installation/setup/ubuntu.html#postgres-database-configuration>`__.
+      the `developer setup instructions <https://opendatacube.readthedocs.io/en/latest/installation/setup/ubuntu.html#postgres-database-configuration>`__.
```

Comment:
Title of subsection has changed. Should now point to https://opendatacube.readthedocs.io/en/latest/installation/setup/ubuntu.html#postgres-testing-database-configuration